packages/netlify-cms-backend-github/src/GraphQLAPI.ts

```diff
@@ -329,8 +329,11 @@ export default class GraphQLAPI extends API {
     const { originRepoOwner: owner, originRepoName: name } = this;
     const { data } = await this.query({ query: queries.statues, variables: { owner, name, sha } });
     if (data.repository.object) {
-      const { status } = data.repository.object;
+      const { status, deployments } = data.repository.object;
       const { contexts } = status || { contexts: [] };
+      if (deployments.nodes[0] && deployments.nodes[0].latestStatus) {
```

Comment:
Similar comment as before, but less critical here since we don't need to send additional requests.
Does it make sense to extract the status both from the commit and the deployment?